services/parser-database/tests/test_connector.py

```diff
@@ -21,18 +22,47 @@
 
 in_memory_value_mock = {
     "ciclista": [
-        {"id": "5", "articleNumber": 5, "frequency": 3},
-        {"id": "45", "articleNumber": 45, "frequency": 1},
-        {"id": "99", "articleNumber": 99, "frequency": 7},
+        {"id": 5, "frequency": 3},
+        {"id": 45, "frequency": 1},
+        {"id": 99, "frequency": 7},
     ],
     "licencia": [
-        {"id": "89", "articleNumber": 89, "frequency": 3},
-        {"id": "45", "articleNumber": 45, "frequency": 3},
-        {"id": "125", "articleNumber": 125, "frequency": 2},
+        {"id": 89, "frequency": 3},
+        {"id": 45, "frequency": 3},
+        {"id": 125, "frequency": 2},
```

Comment:
Why is removing `articleNumber` necessary?

Reply:
this is the keywords_in_memory mock and keywords_in_memory no longer has that attribute as per your newest change.